text/text.md

```diff
@@ -124,19 +131,19 @@ I have a grudge with that, either. It's all for the greater good of the
 Republic, ain't it? Keeping capitalists names out of the Republic and
 putting the names the great spirit of the people gave them. Whatever.
 
-So, sunny it was and I was walking in the shade. OK? Don't say I keep
-repeating myself, you don't stop whining and yawning! Here comes what
+So, it was sunny, and I was walking in the shade. OK? Don't say I keep
+repeating myself, you don't stop whining and yawning!  Here comes what
 happened then: these two dandy types came up to me from the sea
-shore, maybe they had alighted from a ship, maybe from one of the airships that 
+shore. Maybe they had alighted from a ship from one of the airships that 
 had landed a bit farther up the docks. They were dressed to the nines and sweating 
```

Comment:
It's again a running phrase, the guy is ramblihg, repeats himself. An attempt (maybe failed) at colloquialism.